### DISCUSSION TITLE
Question for @SudhanPlayz

### DISCUSSION THREAD

**Discussion by fademaze-no** [SudhanPlayz/Discord-MusicBot#824] [Q&A] [answered]
HOW do you Get a verified domain? Hard and needs money! ;-;

**Comment by JotaroKujo0525** [comment]
Tf

**Reply by fademaze-no** [reply]
Do you want to be a member of CharveCompany?

**Comment by ItzSudhan** [comment, marked as answer]
> HOW do you Get a verified domain? Hard and needs money! ;-;

what do you mean by "verified domain"? and yes you need money lol ;-;

**Reply by fademaze-no** [reply]
github pages to host
mean: (*.github.io) and (*.com, uk, us, net) etc.

**Reply by ItzSudhan** [reply]
.github.io is free
https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site
